cw-orc/src/daemon/queriers.rs

```diff
@@ -1,3 +1,188 @@
+//! # DaemonQuerier
+//!
+//! ## Usage examples
+//!
+//! Note: all following examples need the following code:
+//!
+//! ```rust
+//! use std::sync::Arc;
+//! use tokio::runtime::Runtime;
+//! let rt = Arc::new(Runtime::new().unwrap());
```

Comment:
This doesn't need to be `Arc` for these examples. 
You should tell the developer that he needs `Channel` and that he can get it by either constructing it or by calling an object that implements `ChannelAccess` like the Daemon.